aspnet/signalr/overview/performance/scaleout-with-sql-server.md

```diff
@@ -34,7 +34,7 @@ by [Mike Wasson](https://github.com/MikeWasson), [Patrick Fletcher](https://gith
 
 In this tutorial, you will use SQL Server to distribute messages across a SignalR application that is deployed in two separate IIS instances. You can also run this tutorial on a single test machine, but to get the full effect, you need to deploy the SignalR application to two or more servers. You must also install SQL Server on one of the servers, or on a separate dedicated server. Another option is to run the tutorial using VMs on Azure.
 
-![An image with arrows that go from SQL Server to VMs to computers. One arrow labeled "Update" starts at VMs and goes to SQL Server.](scaleout-with-sql-server/_static/image1.png)
+![Diagram that shows arrows going from S Q L Server to V M to computers One arrow labeled Update starts at V M and goes to S Q L Server.](scaleout-with-sql-server/_static/image1.png)
```

Comment:
Lets split up this sentence too. "Diagram that shows arrows going from S Q L Server to V M to computers. One arrow labeled Update starts at V M and goes to S Q L Server."